Bulldozer.CSV/Maps/Person.cs

```diff
@@ -0,0 +1,913 @@
+
+                switch ( personCsv.RecordStatus )
+                {
+                    case CSVInstance.RecordStatus.Active:
+                        newPerson.RecordStatusValueId = this.RecordStatusDVDict[Rock.SystemGuid.DefinedValue.PERSON_RECORD_STATUS_ACTIVE.AsGuid()]?.Id;
+                        break;
+
+                    case CSVInstance.RecordStatus.Inactive:
+                        newPerson.RecordStatusValueId = this.RecordStatusDVDict[Rock.SystemGuid.DefinedValue.PERSON_RECORD_STATUS_INACTIVE.AsGuid()]?.Id;
+                        break;
+
+                    case CSVInstance.RecordStatus.Pending:
+                        newPerson.RecordStatusValueId = this.RecordStatusDVDict[Rock.SystemGuid.DefinedValue.PERSON_RECORD_STATUS_PENDING.AsGuid()]?.Id;
+                        break;
+                }
+
+                if ( !string.IsNullOrEmpty( personCsv.ConnectionStatus ) )
+                {
+                    newPerson.ConnectionStatusValueId = ConnectionStatusDVDict[personCsv.ConnectionStatus]?.Id;
+                }
+
+                if ( !string.IsNullOrEmpty( personCsv.Salutation ) )
+                {
+                    var prefix = personCsv.Salutation.RemoveSpecialCharacters();
+                    newPerson.TitleValueId = TitleDVDict[prefix]?.Id;
+                }
+
+                if ( !string.IsNullOrEmpty( personCsv.Suffix ) )
+                {
+                    var suffix = personCsv.Suffix.RemoveSpecialCharacters();
+                    newPerson.SuffixValueId = SuffixDVDict[suffix]?.Id;
+                }
+
+                if ( personCsv.Birthdate.HasValue )
+                {
+                    newPerson.BirthMonth = personCsv.Birthdate.Value.Month;
+                    newPerson.BirthDay = personCsv.Birthdate.Value.Day;
+                    newPerson.BirthYear = personCsv.Birthdate.Value.Year == personCsv.DefaultBirthdateYear ? ( int? ) null : personCsv.Birthdate.Value.Year;
+                }
+
+                if ( !string.IsNullOrEmpty( personCsv.MaritalStatus ) )
+                {
+                    newPerson.MaritalStatusValueId = MaritalStatusDVDict[personCsv.MaritalStatus]?.Id;
+                }
+
+                // do a case-insensitive lookup GradeOffset from either the Description ("Kindergarten", "1st Grade", etc) or Abbreviation ("K", "1st", etc)
+                int? gradeOffset = null;
+                if ( !string.IsNullOrWhiteSpace( newPerson.Grade ) )
+                {
+                    gradeOffset = gradeOffsetLookupFromDescription.GetValueOrNull( newPerson.Grade );
+                    if ( gradeOffset == null )
+                    {
+                        gradeOffset = gradeOffsetLookupFromAbbreviation.GetValueOrNull( newPerson.Grade );
+                    }
+                }
+
+                if ( gradeOffset.HasValue )
+                {
+                    newPerson.GraduationYear = Person.GraduationYearFromGradeOffset( gradeOffset );
+                }
```

Comment:
Need to test grade and see why it is not applying to people. Possibly the graduation year from grade offset having issues? Possibly add support for GraduationYear instead of Grade?